site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -74,7 +74,7 @@ More samples can be found in https://github.com/milvus-io/milvus-operator/tree/m
 CRD Documentation can be found in https://github.com/milvus-io/milvus-operator/tree/main/docs/CRD
 ```
 
-#### 2. Install by kubectl command
+### 2. Install by kubectl command
```

Comment:
kubectl is in code format in line 49. Please check if the code format is necessary.

Reply:
[6ee7e70](https://github.com/milvus-io/milvus-docs/pull/1624/commits/6ee7e700ef72105b010b1c6e4a62bbc8e4600dc0)